.github/workflows/continous-delivery.yml

```diff
@@ -189,6 +193,10 @@ jobs:
     if: github.event_name == 'workflow_dispatch'
 
     steps:
+      - uses: actions/checkout@v3.2.0
+        with:
+          persist-credentials: false
```

Comment:
Could we try setting "create_credentials_file" to false in google-github-actions/auth instead of adding this if it works without this step?